content/blog/cheerpj-hackathon.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: "Announcing CheerpJ: The Hackathon!"
+slug: cheerpj-hackathon
+description: |
+  The Java in the browser challenge is coming this autumn.
+authors:
+  - gaby
+  - alessandro
+pubDate: "October 3 2025"
+heroImage: "./cj-hackathon-banner.png"
+featured: true
+tags:
+  - CheerpJ
+---
+
+import LinkButton from "@leaningtech/astro-theme/components/LinkButton.astro";
+import { DISCORD_URL } from "@/consts.ts";
+
+Earlier this year, we were pleased to host the second edition of our now traditional WebVM: The Hackathon. We were elated to see many members of our online community, along with Leeds tech enthusiasts and professionals, participating. Today, we are excited to announce something new: a CheerpJ-themed hackathon, which (as usual) will take place both online and in-person in Leeds!
+
+And guess what? The winning team will receive some Ama$ing prize$!
+
+## Event details
+
+Prep your seatbelts and save the dates!
+
+- **Opening date (hybrid):** Friday, 14/November/2025
+- **Hacking period (online):** Saturday 15 Nov - Sunday 30 Nov
+- **Closing date & submission deadline (hybrid)**: Monday, 1/December/2025
+- **Location:** Online (Discord) and in-person (Nexus, Leeds, UK)
+- **Event website:** https://labs.leaningtech.com/cheerpj-hackathon
+
+## CheerpJ in a nutshell
```

Comment:
Should this section maybe go above the event details? We have not said what CheerpJ is and the reader might have given up already at this point.